common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java

```diff
@@ -336,6 +336,10 @@ public class BrokerConfig extends BrokerIdentity {
 
     private long syncControllerMetadataPeriod = 10 * 1000;
 
+    // It is an important basis for the controller to choose the broker master. Under the same conditions,
```

Comment:
How about making the definition of "higher priority" clearer in the comment or the document? Like is the priority higher while the value of brokerElectionPriority is larger or smaller?

Reply:
Sure